src/Common/HashTable/Hash.h

```diff
@@ -339,6 +352,21 @@ struct UInt256HashCRC32
     }
 };
 
+#elif defined(__aarch64__) && defined(__ARM_FEATURE_CRC32)
+
+struct UInt256HashCRC32
```

Comment:
Will lead to incompatibility of different versions on cluster, I will add a note.